modin/core/execution/unidist/implementations/pandas_on_unidist/io/io.py

```diff
@@ -258,3 +261,66 @@ def func(df, **kw):  # pragma: no cover
         UnidistWrapper.materialize(
             [part.list_of_blocks[0] for row in result for part in row]
         )
+
+    @classmethod
+    def from_map(cls, func, iterable, *args, **kwargs):
```

Comment:
Is it possible to use already implemented functions with `num_splits=1`?

Reply:
I don't quite get what would you like use use instead. Please elaborate. We are adding a new `from_map` by analogy with other io functions.

Reply:
I suppose we can't use anything from existing functionality as every method of a Modin Dataframe assumes there is a dataframe with partitions to apply a function to.